gapsule/handlers/RepoHandler.py

```diff
@@ -1,12 +1,15 @@
 import os
 import json
 import tornado.web
+from gapsule.utils.decorators import ajaxquery
 from gapsule.models.repo import (get_commits_num, get_branches_name, get_releases_num,
                                 get_contributors_info, get_specified_path,
                                 get_file_content)
 
 class CodeListHandler(tornado.web.RequestHandler):
-    def get(self):
+    @ajaxquery
+    def get(self, *args, **kwargs):
```

Comment:
```suggestion
    def get(self, owner, reponame):
```
不是*args,**kwargs，这里的参数是和url的正则表达式对应的。另外建议在正则表达式中使用带命名的形式。

Reply:
这点就是从URL得到的参数